internal/controller/apiextensions/managed/setup.go

```diff
@@ -41,7 +41,7 @@ func Setup(mgr ctrl.Manager, o apiextensionscontroller.Options) error {
 
 	r := NewReconciler(mgr,
 		WithLogger(o.Logger.WithValues("controller", name)),
-		WithRecorder(event.NewAPIRecorder(mgr.GetEventRecorderFor(name), o.EventFilterFunctions...)),
```

Comment:
This was also incorrectly included in the reverted merge PRs. This option doesn't exist in the release-2.0 branch, which is why the build was failing before this PR.